integrated_channels/moodle/client.py

```diff
@@ -178,9 +273,11 @@ def get_course_id(self, key):
         response = self._get_courses(key)
         parsed_response = json.loads(response.text)
         if not parsed_response.get('courses'):
-            raise ClientError('MoodleAPIClient request failed: 404 Course key '
-                              '"{}" not found in Moodle.'.format(key))
-
+            raise ClientError(
+                'MoodleAPIClient request failed: 404 Course key '
+                '"{}" not found in Moodle.'.format(key),
+                HTTPStatus.NOT_FOUND.value
```

Comment:
what actually is the value of Not found here? I'm just not familiar with it.

Reply:
404!

Reply:
oh it's just the status code. They don't use `.code`? Weird

Reply:
I just tried `.code` just to __make sure__ aannnnd got an attribute not found.

Reply:
from their doc:
```>>> HTTPStatus.OK.value
200
>>> HTTPStatus.OK.phrase
'OK'
```